apps/db/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/InntektsmeldingRepositoryTest.kt

```diff
@@ -44,6 +44,84 @@ class InntektsmeldingRepositoryTest :
         val inntektsmeldingRepo = InntektsmeldingRepository(db)
         val testRepo = TestRepo(db)
 
+        context(InntektsmeldingRepository::hentNyesteInntektsmelding.name) {
```

Comment:
Ingen funksjonelle endringener her, bare flytter på funksjoner for å få en logisk gruppering.